src/main/java/de/fearnixx/jeak/service/controller/connection/HttpServer.java

```diff
@@ -0,0 +1,205 @@
+package de.fearnixx.jeak.service.controller.connection;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.fearnixx.jeak.service.controller.controller.ControllerContainer;
+import de.fearnixx.jeak.service.controller.controller.ControllerMethod;
+import de.fearnixx.jeak.service.controller.controller.MethodParameter;
+import de.fearnixx.jeak.service.controller.reflect.RequestBody;
+import de.fearnixx.jeak.service.controller.reflect.RequestMapping;
+import de.fearnixx.jeak.service.controller.reflect.RequestParam;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Route;
+import spark.Spark;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.util.List;
+import java.util.function.Function;
+
+import static spark.Spark.*;
+
+/**
+ * A wrapper for the http server.
+ */
+public class HttpServer {
+    private static final Logger logger = LoggerFactory.getLogger(HttpServer.class);
+    private static final String API_ENDPOINT = "/api";
+    private int port = 8723;
```

Comment:
Please make this configurable :slightly_smiling_face:

Reply:
I would suggest doing this in a seperate issue